code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm

```diff
@@ -93,6 +124,16 @@
 	result = /obj/item/reagent_containers/food/snacks/spidereggsham
 	subcategory = CAT_MEAT
 
+/datum/crafting_recipe/food/sushie_egg
+	name = "Tobiko"
```

Comment:
If you included Tobiko why not include tamago, using eggs you'll actually have access to every round?

Reply:
Their not ment to be round start. Its Exotic